scripts/camViewer

```diff
@@ -258,15 +274,44 @@ fi
 
 if [ $ENABLE -gt 0 ]; then 
     echo "Enabling ${CAMNAME}"
-    IOC=$(iocfpv "${CAMPVFULL}")
-    $imgr "$IOC" --hutch "$hutch" --enable   
+    for ioc in `iocfpv "$CAMPVFULL:IMAGE1"`; do
```

Comment:
Can you briefly explain what these loops are fixing, compared to the previous implementation? I'm not familiar with this `iocfpv` command or the implications of doing this multiple times.

Reply:
Since a lot of the fee cameras have multiple iocs associated with a single PV this loops over all the iocs for a given PV and picks the relevant one.

Reply:
If you mean to say we still have traces of earlier implementations for a given camera, I'd suggest cleaning this up. If that is not what you meant, can you expand on why we have multiple IOCs associated with a given <BASE>:IMAGE1 PV?

Reply:
Yes, I meant to say that we have traces of earlier implementations.